assets/app/view/game/part/city_slot.rb

```diff
@@ -54,7 +54,21 @@ def render_part
             radius -= 4
           end
 
-          children = [h(:circle, attrs: { r: @radius, fill: color })]
+          token_attrs = {
+            r: @radius,
+            fill: color,
+          }
+
+          if @game&.highlight_token?(@token)
+            radius -= 3
+            token_attrs.merge!({
```

Comment:
i don't like the format of this

merge!(
  x: '',
  y'',
)

Reply:
OK